CHANGELOG.md

```diff
@@ -15,6 +15,10 @@
 - Prevent sporadic data loss in `EnvelopeProcessorService`. ([#2454](https://github.com/getsentry/relay/pull/2454))
 - Prevent panic when android trace contains invalid start time. ([#2457](https://github.com/getsentry/relay/pull/2457))
 
+**Features**:
```

Comment:
There's a duplicate `Features` section, let's move the entry up to the existing one.